tests/test_framerate_regulator.py

```diff
@@ -58,3 +63,15 @@ def test_sleep():
 
     assert regulator.called == 200
     assert after - before >= 2.0
+
+
```

Comment:
Agreed - let's do it as a separate PR. If count is zero or start time is None, both those methods could return None